docs/cloud/netdata-functions.md

```diff
@@ -41,16 +50,16 @@ concern is to ensure no sensitive
 information or disruptive actions are exposed through the Agent's API.
 
 With the communication between the Netdata Agent and Netdata Cloud being
-through [ACLK](https://github.com/netdata/netdata/blob/master/docs/concepts/netdata-agent/aclk.md#ACLK) this
+through [ACLK](https://github.com/netdata/netdata/blob/master/aclk/README.md) this
 concern is addressed.
 
 ## Related Topics
 
 ### **Related Concepts**
 
-- [ACLK](https://github.com/netdata/netdata/blob/master/docs/concepts/netdata-agent/aclk.md)
-- [plugins.d](https://github.com/netdata/netdata/tree/master/collectors/plugins.d)
+- [ACLK](https://github.com/netdata/netdata/blob/master/aclk/README.md)
+- [plugins.d](https://github.com/netdata/netdata/blob/master/collectors/plugins.d/README.md)
 
 ### Related Tasks
 
-- [Run-time troubleshooting with Functions](docs/nightly/tasks/operations/runtime-troubleshootting-with-function)
+- [Run-time troubleshooting with Functions](https://github.com/netdata/netdata/blob/master/docs/nightly/operations/run-time-troubleshooting-with-functions.md)
```

Comment:
This link is 404. Is it expected?

Reply:
@ilyam8 I tested this after running the ingest flow locally and launching learn and this is working

https://user-images.githubusercontent.com/82235632/215470570-e9348bcc-7662-470d-8a0b-d47f9e9ded25.mp4

Reply:
seems this was referencing some existing file, from a previous merge, that is also getting updated 🤷 
I'm committing a fix

Reply:
https://github.com/netdata/netdata/pull/14331/commits/49570927c96a9451087c2627983e53258a820738